python/tank/platform/qt/login.ui

```diff
@@ -0,0 +1,228 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>LoginDialog</class>
+ <widget class="QDialog" name="LoginDialog">
+  <property name="windowModality">
+   <enum>Qt::ApplicationModal</enum>
+  </property>
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>374</width>
+    <height>324</height>
+   </rect>
+  </property>
+  <property name="styleSheet">
+   <string notr="true">QWidget
```

Comment:
As agreed offline, we’ll hold off doing any cosmetic changes at the moment. The important part it to get the workflow in place.